src/client/sandbox/native-methods.js

```diff
@@ -163,6 +163,7 @@ class NativeMethods {
         this.Blob             = win.Blob || mock;
         this.XMLHttpRequest   = win.XMLHttpRequest || mock;
         this.Image            = win.Image || mock;
+        this.Function         = win.Function || mock;
```

Comment:
@churkin 
Do you remember why we need this instruction?
I remove this and run qunit tests in IE11 and Chrome. All tests are green.